src/sentinel.c

```diff
@@ -4127,7 +4127,7 @@ NULL
 
 void addInfoSectionsToDict(dict *section_dict, char **sections);
 
-/* SENTINEL INFO [section] */
+/* INFO [section [section ...]] */
```

Comment:
```suggestion
/* SENTINEL INFO [section [section ...]] */
```

Reply:
Sorry, Binbin, the comment INFO [section [section ...]]. When you run in the client side of sentinel, you should not enter "sentinel", otherwise you will get an error message

![image](https://user-images.githubusercontent.com/51993843/165530576-3c9ff9fd-8687-4f4b-b702-ff1bac8b8462.png)